irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -67,57 +108,63 @@ namespace {
 
   std::string checkAccountRolePermission(
       shared_model::interface::permissions::Role permission,
-      const std::string &account_alias = "role_account_id") {
+      const std::string &account_alias) {
```

Comment:
std::string -> shared_model::interface::types::AccountIdType
also, why "alias" instead of "id"?